### ISSUE TITLE
Improve notifications

### ISSUE THREAD

**Issue by jenniferward** [rism-digital/muscat#777]
I have a couple suggestions to improve notifications: 

1. Add  lib_siglum to this part of the "How to use notifications" help text:
Currently the notification system is available only for Sources and for the following fields:
**std_title** Standardized title
**composer** Composer full name
**title** Title on source
**shelf_mark** Library shelf mark
**lib_siglum** Library siglum

2. It would be nice to be able to combine search fields in an alert
Example: lib_siglum: GB-Lbl &  shelf_mark Add. 46125

<strike>3. In the "Source modification report", the link to Muscat is helpful, but I would like a bit more contextual information, such as
-Is the source new, or was it modified?
-Can you include a brief view of the record, i.e. include RISM number, siglum, shelfmark, composer, title. </strike>

**Comment by xhero** [comment]
Partially done in 6.1, the rest for 6.2

**Comment by xhero** [comment]
I implemented a simple query format (8ab5ed7). The syntax is:

```
[model] property:pattern [property:pattern]
```

If ```model``` is not provided the default is Source. For example

```
source lib_siglum:CH* composer:Bach* std_title:*Fugues*
work composer:Mozart*
```
